packages/catalog-process/test/db.test.ts

```diff
@@ -1380,6 +1424,20 @@ describe("database test", async () => {
           createdAt: new Date(Number(writtenPayload.eService?.createdAt)),
         };
         expect(writtenPayload.eService).toEqual(toEServiceV1(expectedEService));
+        expect(copy).toHaveBeenCalledWith(
+          config.storageContainer,
+          config.eserviceDocumentsPath,
+          descriptorInterface.path,
+          expectedInterface.id,
+          descriptorInterface.name
+        );
+        expect(copy).toHaveBeenCalledWith(
+          config.storageContainer,
+          config.eserviceDocumentsPath,
+          document.path,
+          expectedDocument.id,
+          document.name
+        );
```

Comment:
To avoid a similar issue, I am adding a similar check inside the test of the `clone descriptor`, because that operation has to copy both `interface` and `docs`.